candid/evm_rpc.did

```diff
@@ -81,6 +81,16 @@ type HttpOutcallError = variant {
 type InstallArgs = record {
   demo : opt bool;
   manageApiKeys : opt vec principal;
+  logMessageFilter : opt LogMessageFilter;
+};
+type LogMessageFilter = variant {
+  ShowAll;
```

Comment:
Why is `ShowAll` a separate variant but there is no `ShowNone`?

Reply:
How about `IgnoreAll` and `IgnoreOnly(filters)` in addition to `ShowAll` and `ShowOnly(filters)`?

Reply:
Yeah, that works too!

Reply:
Have you maybe thought of instead of having these various (fixed) variants to have simply the filter in the form of a regex? I have the impression this would simplify the design while allowing for greater flexibility. WDYT?

Reply:
I like your suggestion! I have the impressions that brings a lot of flexibility/usability without complexity on our side so I would favour that approach over a fixed list of enum variants as you initially had.